src/bot/ignorellist.ts

```diff
@@ -0,0 +1,4 @@
+export function isIgnored (account: string): boolean {
+  const ignoreList: string[] = (process.env.FAUCET_IGNORE_LIST ?? '').replace(/\s/g, '').split(',') ?? [];
+  return ignoreList.includes(account);
```

Comment:
One should also potentially convert the inputted address for the specific ss58 encoding of the chain the faucet is run for. 

https://polkadot.js.org/docs/keyring/start/ss58#how-is-the-encoding-done

I think someone can just input a different ss58 representation of the address to get around the ignore list.

Reply:
I think this can be done by:

```javascript
import { decodeAddress } from '@polkadot/keyring';

export function isIgnored (account: string): boolean {
    try {
      decodeAddress(arg0);
    } catch (e) {
      logger.warn(`${account} is not in the proper format.`);
      return false;
    }
  const ignoreList: string[] = (process.env.FAUCET_IGNORE_LIST ?? '').replace(/\s/g, '').split(',') ?? [];

  return ignoreList.includes(decodeAddress(account));
}
```

Reply:
This of course will then assume that the ignore list is ss58 specific for the given chain.

Reply:
Yes, this PR aism at a simple solution to have at least something in place.
Since we rely on matrix accounts and those are (slightly) harder to produce than a new address, a filter on the address could be done however a matrix user can also simply be banned from the faucet channel.

Reply:
I added your suggestion. Totally untested yet.

Reply:
I switched to ignoring Matrix accounts, it ended up easier and this is stronger.